references/detection/train.py

```diff
@@ -35,12 +36,17 @@ def plot_samples(images, targets):
         img *= 255
         img = tf.cast(tf.clip_by_value(tf.round(img), 0, 255), dtype=tf.uint8).numpy()
 
-        target = np.zeros(img.shape[:2], dtype=bool)
+        target = np.zeros(img.shape[:2])
         boxes = targets[idx]['boxes'][np.logical_not(targets[idx]['flags'])]
         boxes[:, [0, 2]] = boxes[:, [0, 2]] * img.shape[1]
         boxes[:, [1, 3]] = boxes[:, [1, 3]] * img.shape[0]
         for box in boxes.round().astype(int):
-            target[box[1]: box[3] + 1, box[0]: box[2] + 1] = True
+            if rotation:
+                target[box[1]: box[3] + 1, box[0]: box[2] + 1] = 1
+            else:
+                box = cv2.boxPoints(((box[0], box[2]), (box[1], box[3]), box[4]))
+                box = np.int0(box)
+                cv2.drawContours(target, [box], 0, (255, 255, 255), 2)
```

Comment:
Could you add comments or keywords for `drawContours` ?
In the other case, we were filling with ones, I don't see any `1` here, probably 0 or 2